tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -187,11 +188,14 @@ def fe_input_user_data_repack(
     """
     _input_shapes = []
     _input_names = []
+    model_inputs = input_model.get_inputs()
+    if framework == "pytorch":
+        model_inputs = model_inputs[1:]
```

Comment:
several questions:
1. what is the first input?
2. why can we exclude this input in PT FE internally if it is not used at all? So MO code should not care of it.

Reply:
We should filter inputs on PT FE side. There are two possible solutions: short term (1) apply the same hack on PT FE side (just remove the first input), long term (2) restore model topology and report back the right number of inputs. To avoid blocking at this moment we can apply (1) and plan (2) as a separate task.

Reply:
Exactly, now the solution from PT FE is incomplete.